.github/dependabot.yml

```diff
@@ -5,7 +5,7 @@ updates:
     schedule:
       interval: "weekly"
 
-  - package-ecosystem: "docker"
+  - package-ecosystem: "pip"
```

Comment:
what does this do?

Reply:
Tells dependabot it's a pip library, not a container.

Reply:
I don't really know what dependabot does with that information, but it's correct.